### PR TITLE
Added fixed width to modal

### PR DESCRIPTION
**Pull request by CarloPalinckx**
### This PR:

proposed release: `patch`

**Adds** ✨
- Fixed width to the modal to allow for percentual styling inside of it.

### CODE REVIEW

File: src/components/Modal/style.tsx [hunk 1]
```diff
@@ -19,25 +19,28 @@ const StyledModalWrapper = withProps<ModalWrapperPropsType>(styled.div)`
...
+        max-height: 100%;
+    }
 `;
 
 const StyledModal = styled.div`
-    margin: auto;
-    max-width: 600px;
-    width: calc(100% - 24px);
+    max-width: 100%;
+    width: 791px;
```
Comment (woutsluiter) [inline src/components/Modal/style.tsx]: Peculiar value 😄

File: src/components/Modal/style.tsx [hunk 2]
```diff
@@ -19,25 +19,28 @@ const StyledModalWrapper = withProps<ModalWrapperPropsType>(styled.div)`
...
     flex-grow: 1;
```
Comment (nicokoelewijn) [inline src/components/Modal/style.tsx]: flex-grow can be removed

File: src/components/Modal/style.tsx (hunk 1 above)
Comment (CarloPalinckx) [reply src/components/Modal/style.tsx]: Oops . this should be 792 because ⤵️

Comment (CarloPalinckx) [reply src/components/Modal/style.tsx]: screen shot 2018-07-30 at 09 46 52

File: src/components/Modal/story.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import Modal from '.';
 import Button from '../Button';
 import ButtonGroup from '../ButtonGroup';
 import Text from '../Text';
+import Box from '../Box';
```
Comment (woutsluiter) [inline src/components/Modal/story.tsx]: Unused

**Pull request merged**